user_type.go

```diff
@@ -19,13 +34,13 @@ const (
 	// Corporate user type
 	Corporate UserType = 3
```

Comment:
It looks like they have some different names for these user types now. `2` is now referred to as `Licensed` and `3` is "On-prem" (let's use`OnPrem`). Will you please update these to reflect the current names?